Blackjack/Game/BlackjackGame.m

```diff
@@ -0,0 +1,246 @@
+//
+//  BlackjackGame.m
+//  Blackjack
+//
+//  Created by Alex on 28/09/2022.
+//
+
+#import "BlackjackGame.h"
+#import "PlayingCard.h"
+
+// MARK: - Private Properties
+
+@interface BlackjackGame()
+@property (nonatomic, strong) Deck *deck;
+@property (nonatomic) NSInteger cardsAmountToWin;
+@property (nonatomic) NSInteger dealerMinimumCardEvaluation;
+@property (nonatomic) enum State gameState;
+@property (nonatomic) NSArray *gameStates;
+@end
+
+// MARK: - Implementation
+
+@implementation BlackjackGame
+
+@synthesize dealer = _dealer; // because we provide setter AND getter
+
+enum State {
+    collectBets, dealCards, awaitingPlayersDecision, awaitingDealerDecision
```

Comment:
`Enums` are always with CAPS_SNAKE_CASE.

Also, see NS_ENUM for a better way to write enums in ObjC

Reply:
New line for each enum case

Reply:
Done